app/views/tag/index.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= javascript_include_tag('async_tag_subscriptions.js') %>
```

Comment:
Not sure why it doesn't like this - same as here, no? 

https://github.com/publiclab/plots2/blob/ff7d91eb00fddabfd9169530257cb223e1846ac5/app/views/questions/show.html.erb#L1

Reply:
OH, you know what? Maybe it wants to precompile it in its own file, to minify it, and since we aren't referencing it in application.js, it's not getting merged with everything else. So you could follow the instructions and:

```
        Add `Rails.application.config.assets.precompile += %w( async_tag_subscriptions.js )` to `config/initializers/assets.rb` and restart your server
```

Sound right?